store/src/java/com/zimbra/cs/imap/ImapPath.java

```diff
@@ -560,6 +560,10 @@ boolean isSelectable() throws ServiceException {
 
     boolean isVisible() throws ServiceException {
         boolean isMailFolders = Provisioning.getInstance().getLocalServer().isImapDisplayMailFoldersOnly();
+        if(folder == null) {
+            ZimbraLog.imap.error("ImapFolderStore is null. This session was likely terminated.");
```

Comment:
Would this be more useful as a 'debug' level instead since it is likely to occur in production and isn't truly an 'error' per se?